### DISCUSSION TITLE
Can't update server

### DISCUSSION THREAD

**Discussion by alexcroce** [immich-app/immich#13628] [Q&A] [answered]
### The bug

As soon as i update the server (running in a container in proxmox) the server become unreachable, and the only way to use it again is to delete the container and restore a previous backup

### The OS that Immich Server is running on

Proxmox LXC

### Version of Immich Server

1.116

### Version of Immich Mobile App

1.118

### Platform with the issue

- [X] Server
- [ ] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
.
```

### Your .env content

```Shell
.
```

### Reproduction steps

1. update the server
2. try to reach it

**Comment by danieldietzler** [comment, marked as answer]
Read the release notes.

**Reply by alexcroce** [reply]
Ok, it was my fault. Now it works. Thank you so much!

**Reply by TechSupport26** [reply]
I did the same thing too.

I saw the app said that there was an update, so I went to the web version and that said there was an update so I just updated. I don't remember if it said anything about reading the changes before but I should learn to not do things just because someone or something says to do so.

I am glad that I could just go back to the previous version by changing the .env file and that all I needed to change was the ports in the docker-compose.yml file.
